app/org/sagebionetworks/bridge/models/accounts/User.java

```diff
@@ -90,11 +89,11 @@ public void setHealthCode(String healthCode) {
     }
 
     public String getEncryptedHealthCode() {
-        return encryptor.encrypt(healthCode);
+        return ENCRYPTOR.encrypt(healthCode);
     }
 
     public void setEncryptedHealthCode(String healthCode) {
-        this.healthCode = encryptor.decrypt(healthCode);
+        this.healthCode = ENCRYPTOR.decrypt(healthCode);
```

Comment:
I know what's going on here, this class is User. It's not a managed bean. It could be a prototype bean, but then we'd have to know about spring's application context to instantiate a new user.